applications/launchpad/gui-react/src/components/TBot/TBotPrompt/MessageBox.tsx

```diff
@@ -15,9 +20,13 @@ const MessageBox = (
   {
     animate,
     children,
+    skipButton,
+    $floating,
   }: {
     animate: boolean
     children: ReactNode
+    skipButton?: boolean
+    $floating?: boolean
```

Comment:
No need to use `$` here. The `$` is used with styled components, and it means that styled component won't add  specific prop to the DOM. So here we can use just `floating`... (see next comment)